src/java/main/org/apache/zookeeper/server/command/FourLetterCommands.java

```diff
@@ -153,13 +159,50 @@
      */
     public final static int telnetCloseCmd = 0xfff4fffd;
 
-    final static HashMap<Integer, String> cmd2String =
-        new HashMap<Integer, String>();
+    private static final String ZOOKEEPER_4LW_COMMANDS_WHITELIST = "zookeeper.4lw.commands.whitelist";
+
+    // A property only used in tests to turn on / off entire set of supported four letter word commands.
+    private static final String ZOOKEEPER_4LW_TEST = "zookeeper.test.4lw.enabled";
```

Comment:
I really like this property as it saves me tons of work - but I'll see what I can do.